src/events/components/SearchModule/search.less

```diff
@@ -21,12 +95,13 @@
     }
 
     .DayPickerInput-Overlay {
-      width: 550px;
+      width: 525px;
+      .owCard();
     }
     /* stylelint-enable */
   }
 }
 /* stylelint-disable-next-line selector-type-case, selector-type-no-unknown */
-.inputFromToTo :global DayPickerInput-Overlay {
+.inputFromToTo :global daypickerinput-overlay {
```

Comment:
Why is this changed? As far as I can remember (and I put this here) the `DayPickerInputOverlay`-class is something we can't rename (it is a part of the ReactDatePicker CSS, which we cannot set ourselves), so you are just making the rule useless, and then it should be removed altogether

Reply:
Might have been changed by `yarn lint:fix` ? 

I can't remember changing myself:P